apps/prairielearn/src/middlewares/clientFingerprint.ts

```diff
@@ -18,11 +36,6 @@ export default asyncHandler(async (req, res, next) => {
 
   const params = {
     ip_address: req.ip,
-    // We are passing the authn user id here. However, we are checking
-    // in the SQL query 'update_assessment_instance_fingerprint'
-    // that the authn user is the owner of the assessment. This will keep us
-    // from inadvertently recording a fingerprint change
-    // for an instructor viewing the assessment instance.
```

Comment:
This is a useful comment; could we keep it around somewhere?

Reply:
Yes, this was lost in the process of refactoring functions to reduce duplication. I have now restored this comment to the same place in 52d0e1a.